go.mod

```diff
@@ -1,16 +1,17 @@
 module github.com/GoogleCloudPlatform/spanner-migration-tool
 
-go 1.19
+go 1.21
```

Comment:
this may lead to some fan out on the rest of SMT. Worth looking into during testing.

Reply:
Yes this would require changes in the kokoro job first to ensure that the releases don't break